### PR TITLE
[NEW BONUS GUIDE] Wireguard VPN

### PR DESCRIPTION
**Pull request by twofaktor**
#### What

This PR proposes a new bonus guide to securely and privately remote access your MiniBolt node through the Wireguard server installed on MiniBolt and Wireguard client on your clients.

#### Why

We seek to connect to our own server without exposing our real IP to the rest of the network for security and privacy reasons, and at the same time to avoid using Tor for latency reasons

#### How

A complete guide explaining step by step how to build it

#### Scope

- [ ] significant change to core configuration
- [X] independent bonus guide
- [ ] simple bug fix

### CODE REVIEW

**Comment by 1ma** [comment]
@twofaktor completed & ready to review.

**Comment by 1ma** [comment]
> Why
>
> We seek to connect to our own server without exposing our real IP to the rest of the network for security and privacy
> reasons, and at the same time to avoid using Tor for latency reasons

WireGuard doesn't do anything to try to hide your IP, like Tor. But it makes it possible to access your MiniBolt services from outside your home network without having to open their ports to the internet.

WireGuard's traffic itself is encrypted, but it can be seen going to and from your real IP address.

File: guide/bonus/system/wireguard-VPN.md [hunk 1]
```diff
@@ -126,15 +126,15 @@ Public key ***example expected output***, we'll it call `"<Your_Server_Public_Ke
...
+Take note and securely back up this public key in your preferred password manager (Bitwarden, Lastpass, Keypass...).
 
 * Edit `wg0.conf` file
 
   ```sh
   $ sudo nano /etc/wireguard/wg0.conf
   ```
 
-Write the following and replace `"<Your_Server_Private_Key>"` with the last obtained output:
+Write the following contain and replace only `"<Your_Server_Private_Key>"` with the last obtained output:
```
Comment (1ma) [inline guide/bonus/system/wireguard-VPN.md:137]: contain -> content

File: guide/bonus/system/wireguard-VPN.md [hunk 2]
```diff
@@ -181,10 +181,9 @@ You'll see the following form:
 ![Registration Page](../../../images/desec_io1.png)
 
 Here you must select the second option (Register a new domain under dedyn.io) and your desired subdomain name.
-For the purposes of this demo I've generated a random string `"yoursubdomain"`, but ***you can use anything memorable*** to you as long as no one has already taken that name.
+For the purposes of this demo I've type a random subdomain `"yoursubdomain"`, but ***you can use anything memorable*** to you as long as no one has already taken that name.
```
Comment (1ma) [inline guide/bonus/system/wireguard-VPN.md:184]: type -> typed

**Review by 1ma — CHANGES_REQUESTED** [review]

**Comment by twofaktor** [comment]
I merged this PR with changes of other topics wrongly, migrated to #38

**Pull request closed without merging**